README.md

```diff
@@ -57,23 +58,22 @@ bin/cake server
 
 Note that this should never be used for a production site!
 
-### Experimental Docker Alternative
+### Docker Quick Start (Experimental!)
 
-The included Dockerfile and docker-compose.yaml files are new and will hopefully make it quicker and easier to rapidly test out the application on your own hardware.  **The Docker implementation should be considered experimental until further testing can be completed.**  If you have any issues or suggestions for simplifying deployment with Docker, or encounter any issues with deploying Zuluru under Docker, please create an issue here or at the https://github.com/pl77/Zuluru3 docker fork so it can be investigated.  
+Run `docker-compose pull && docker-compose up -d` and open
+http://localhost/installer/install. The database credentials are `host: db`,
+`username: zuluru`, `password: userpassword`. This provides a quick and easy
```

Comment:
Can the database configuration be automated or at least pre-populated to eliminate this step?

Reply:
I stumbled over this as well.  I'm not overly familiar with php so I wasn't going to attempt to replicate those steps in bash.